### PR TITLE
[WIP-test] Lab color space

### PR DESCRIPTION
**Pull request by cceyda**
Testing #350 https://github.com/kornia/kornia/pull/350

### CODE REVIEW

File: kornia/color/lab.py [hunk 1]
```diff
@@ -194,14 +191,13 @@ def __init__(self) -> None:
 
         self.xyz_to_rgb = torch.tensor([
             #     r           g          b
-            [ 3.24048134, -0.96925495,  0.05564664], # x
-            [-1.53715152,  1.87599, -0.20404134], # y
-            [-0.49853633,  0.04155593,  1.05731107], # z
+            [3.24048134, -0.96925495, 0.05564664],  # x
+            [-1.53715152, 1.87599, -0.20404134],  # y
+            [-0.49853633, 0.04155593, 1.05731107],  # z
```
Comment (edgarriba) [inline kornia/color/lab.py:196]: check this out: https://kornia.readthedocs.io/en/latest/color.html#kornia.color.xyz_to_rgb

File: kornia/color/lab.py [hunk 2]
```diff
@@ -259,45 +254,44 @@ def cielab_to_xyz(self, input: torch.Tensor) -> torch.Tensor:
...
+        xyz_pixels = (3 * epsilon**2 * (fxfyfz_pixels - 4 / 29)) * linear_mask + (fxfyfz_pixels ** 3) * exponential_mask
 
         # denormalize for D65 white point
         xyz_denormalization_const = self.xyz_denormalization_const.type(input.data.type())
         xyz_pixels = torch.mul(xyz_pixels, xyz_denormalization_const)
 
         return xyz_pixels
 
-
     def xyz_to_srgb(self, input: torch.Tensor) -> torch.Tensor:
```
Comment (edgarriba) [inline kornia/color/lab.py:269]: this could be also an separated core function

**Review by edgarriba — COMMENTED** [review]
@cceyda thanks  for taking this ! not sure the rebase has been done properly. What I would suggest is to start a fresh branch and as I suggest decompose into small subroutines. Will be easier to test and will add more value to the library by having extra color core functions. We can guide you in case you need more help.

**Comment by cceyda** [comment]
> @cceyda thanks for taking this ! not sure the rebase has been done properly. What I would suggest is to start a fresh branch and as I suggest decompose into small subroutines. Will be easier to test and will add more value to the library by having extra color core functions. We can guide you in case you need more help.

yes I haven't finished rebasing yet was just playing around with it in github-codespaces (then other things got in the way) but will look at it soon. Thank you for the suggestions

**Comment by edgarriba** [comment]
sure! I'll tag this for v0.5 (in a month or so)

**Comment by cceyda** [comment]
Abandon for https://github.com/kornia/kornia/pull/712

**Pull request closed without merging**